statuspage/resource_component.go

```diff
@@ -124,7 +125,7 @@ func resourceComponent() *schema.Resource {
 			},
 			"only_show_if_degraded": &schema.Schema{
 				Type:        schema.TypeBool,
-				Description: "Requires a special feature flag to be enabled",
+				Description: "Should this component be shown component only if in degraded state",
```

Comment:
How about "Only show this component if in degraded state" ?

Reply:
Sounds good.

Reply:
Sounds good.